CARDS-2064 - Add support for formatting in the question text

marta- opened the pull request:
Testing: in the questionnaire wizard, create a question with simple markdown formatting in the text, e.g "This is an _important_ question".

marta- committed:
CARDS-2064 - Add support for formatting in the question text

sdumitriu reviewed (approved):
Works well when viewing/editing forms.

In the questionnaire editor, should the title be formatted?

![image](https://user-images.githubusercontent.com/88663/217908697-76e28a49-6eb8-478b-bc95-91170a5c23e0.png)

marta- commented:
> In the questionnaire editor, should the title be formatted?

Right, yes, will do.

marta- committed:
CARDS-2064 - Add support for formatting in the question text
Addressing review comments: the question text should now be displayed as formatted in the questionnaire wizard's edit mode

sdumitriu commented:
The title is not formatted in the card header, e.g.:

![image](https://user-images.githubusercontent.com/88663/217915175-079c93cb-dba5-4270-abe7-81001b49333c.png)

but I think that's OK.

marta- commented:
> The title is not formatted in the card header

Let me take care of that too

marta- committed:
CARDS-2064 - Add support for formatting in the question text
Addressing review comments: the titles of question cards should be formatted in the questionnaire qizard's edit mode

marta- committed:
CARDS-2064 - Add support for formatting in the question text
Bonus: also allow the section label to be formatted

marta- commented:
> > The title is not formatted in the card header
> 
> Let me take care of that too

Done in 9a9e91b.

Also enabled formatting for section labels in 8c6bcec.

sashaandjic reviewed (approved):
tested
![image](https://user-images.githubusercontent.com/23243244/217997796-c7cf83c4-a107-42ae-8e57-70545b8b9e5f.png)
